src/app/portlets/content-types/form/content-types-form.component.spec.ts

```diff
@@ -31,21 +31,21 @@ describe('ContentTypesFormComponent', () => {
     beforeEach(async(() => {
 
         let messageServiceMock = new MockMessageService({
-            'Detail-Page': 'Detail Page',
-            'Expire-Date-Field': 'Expire Date Field',
-            'Host-Folder': 'Host or Folder',
-            'Identifier': 'Identifier',
+            'contenttypes.form.field.detail.page': 'Detail Page',
+            'contenttypes.form.field.expire.date.field': 'Expire Date Field',
+            'contenttypes.form.field.host.folder': 'Host or Folder',
+            'contenttypes.form.identifier': 'Identifier',
             'Properties': 'Properties',
```

Comment:
What about this one?

Reply:
Right, those keys were no longer used. I removed them in the latest commit.